packages/scripts/src/create-webpack-configs.ts

```diff
@@ -205,3 +206,76 @@ function createWebpackConfig({
         plugins: [...basePlugins, ...plugins],
     };
 }
+
+export function createWebpackConfigForExteranlFeature({
+    baseConfig,
+    target,
+    enviroments,
+    virtualModules,
+    features,
+    context,
+    mode = 'development',
+    outputPath,
+    plugins = [],
+    entry = {},
+    featureName,
+}: ICreateWebpackConfigOptions): webpack.Configuration {
+    const externals: Record<string, string> = {
+        '@wixc3/engine-core': 'EngineCore',
+    };
+    for (const feature of [...features.values()]) {
+        if (feature.isRoot && (!featureName || featureName === feature.scopedName)) {
+            for (const [envName, childEnvs] of enviroments) {
+                const entryPath = fs.join(context, `${envName}.js`);
+                entry[envName] = entryPath;
+                const publicPathParts = [EXTERNAL_FEATURES_BASE_URI, feature.packageName];
+                if (feature.scopedName !== feature.name) {
+                    publicPathParts.push(feature.name);
+                }
+                publicPathParts.push(basename(outputPath) + '/');
+                virtualModules[entryPath] = createExternalBrowserEntrypoint({
+                    ...feature,
+                    childEnvs,
+                    envName,
+                    publicPath: join(...publicPathParts),
+                    loadStatement: webpackImportStatement,
+                    target: target === 'webworker' ? 'webworker' : 'web',
+                });
+            }
+        } else {
+            if (feature.packageName !== '@wixc3/engine-core') {
+                const featureFilePath = remapFileRequest(feature);
+                externals[featureFilePath] =
```

Comment:
collecting all external features, and referencing them to the object defined earlier on the global context